package.json

```diff
@@ -62,34 +58,33 @@
     "moment": "^2.24.0",
     "pluralize": "^8.0.0",
     "prettier": "^1.18.2",
-    "raw-loader": "^3.1.0",
     "requirejs": "^2.3.6",
+    "rollup": "^1.26.0",
+    "rollup-plugin-alias": "^2.2.0",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-string": "^3.0.0",
+    "rollup-plugin-terser": "^5.1.2",
     "serve": "^11.2.0",
-    "text": "github:requirejs/text#d04de4ffd7bf5ba6cb80cdca2d40d4f6f52a1b1f",
     "typescript": "^3.6.4",
-    "webidl2": "^23.10.1",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.3.9"
+    "webidl2": "^23.10.1"
   },
   "scripts": {
-    "babel:build": "babel src -d js --source-maps -q",
-    "babel:watch": "babel src -d js --watch --source-maps",
-    "build:components": "npm run copydeps && npm run babel:build",
     "build:geonovum-debug": "npm run build:geonovum -- --debug",
     "build:geonovum": "node ./tools/builder.js --profile=geonovum",
     "build:w3c-debug": "npm run build:w3c -- --debug",
     "build:w3c-with-jquery": "node ./tools/builder.js --profile=w3c-common",
     "build:w3c": "node ./tools/builder.js --profile=w3c",
-    "build": "snyk protect && npm run build:components",
-    "copydeps": "node ./tools/copydeps.js",
+    "build": "snyk protect",
+    "builddeps": "rollup -c js/deps/rollup.config.js",
     "karma": "karma start --single-run",
     "lint": "tsc -p src/jsconfig.json && eslint .",
-    "prepare": "npm run snyk-protect && npm run build:components",
+    "prepare": "npm run snyk-protect && npm run builddeps",
     "release": "node ./tools/release.js",
     "server": "npm start",
     "snyk-protect": "snyk protect",
     "snyk": "snyk",
-    "start": "npm run build:components && serve",
+    "start": "serve",
```

Comment:
This is where the "dev server" would come in... basically, what we want here is for rollup to be watching files and automatically running babel for us.

Reply:
oh, and also automatically building the "development" bundle.

Reply:
https://rollupjs.org/guide/en/#rollupwatch

Reply:
We don't need babel anymore here, so you just want to refresh your browser and that's all. (Only if you are using non-built version)